libreport.spec.in

```diff
@@ -320,7 +320,7 @@ CFLAGS="%{optflags} -Werror" %configure \
 %if %{without bugzilla}
         --without-bugzilla \
 %endif
-%if 0%{?rhel:%{rhel} < 7}%{?fedora:%{fedora} < 18}
+%if 0%{?rhel} < 7
```

Comment:
On Fedora, this condition equals `"if 0 < 7"` which is of cores true, but we don't want to pass the arguments below to the ./configure script on Fedora.